pkg/kubelet/cadvisor.go

```diff
@@ -24,6 +25,17 @@ import (
 	cadvisor "github.com/google/cadvisor/info"
 )
 
+var (
+	// ErrNoKubeletContainers returned when there are not containers managed by the kubelet (ie: either no containers on the node, or none that the kubelet cares about).
+	ErrNoKubeletContainers = errors.New("No containers managed by kubelet")
```

Comment:
nit: errors must start with lower case here and elsewhere. If this is not strictly enforced across the codebase you can ignore this comment.